gateleen-playground/src/main/java/org/swisspush/gateleen/playground/Server.java

```diff
@@ -181,13 +181,15 @@ public void start() {
 
         String redisHost = (String) props.get("redis.host");
         Integer redisPort = (Integer) props.get("redis.port");
+        Boolean redisEnableTls = (Boolean) props.get("redis.enableTls");
 
         props.put(ExpansionHandler.MAX_EXPANSION_LEVEL_HARD_PROPERTY, "100");
         props.put(ExpansionHandler.MAX_EXPANSION_LEVEL_SOFT_PROPERTY, "50");
 
         RunConfig.deployModules(vertx, Server.class, props, success -> {
             if (success) {
-                redisClient = new RedisClient(vertx, new RedisOptions().setConnectionString("redis://" + redisHost + ":" + redisPort));
+                String protocol = redisEnableTls ? "rediss://" : "redis://";
```

Comment:
WARN: if property is not set, we will run into an NPE here.

Reply:
check added when read property